code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -867,7 +906,7 @@ var/list/robot_verbs_default = list(
 		return
 
 /mob/living/silicon/robot/verb/unlock_own_cover()
-	set category = "Robot Commands"
+	set category = "Robot Commands" //Changed to Robot Commands because Drones are robots and having a separate tab for this is retarded.

```

Comment:
Thank you for your incredibly helpful comment here. It needs to be removed. It wasn't even changed.